.ci/register_package.jl

```diff
@@ -131,5 +132,6 @@ mktempdir() do download_dir
     end
 end
 
-# Sub off to Registrator to create a PR to General
-BinaryBuilder.register_jll(name, build_version, dependencies, julia_compat; lazy_artifacts=lazy_artifacts)
+# Sub off to Registrator to create a PR to General.  Note: it's important to pass both
+# `augment_platform_block` and `lazy_artifacts` to build the right Project dictionary
+BinaryBuilder.register_jll(name, build_version, dependencies, julia_compat; augment_platform_block, lazy_artifacts)
```

Comment:
@vchuravy @maleadt we were missing to explicitly pass `augment_platform_block` here: the project dictionary was built incorrectly (without TOML as dependency and the right JLLWrappers compat), and the PR to the General registry was wrong: https://github.com/JuliaRegistries/General/pull/57117#issuecomment-1075792098.  But as far as I can tell, with https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/1191 all is good (and we also have tests!)

Reply:
Great!